ch04packaging/03Packaging.ipynb.py

```diff
@@ -362,21 +404,18 @@ def process():
 [project.scripts]
 greet = "greetings.command:process"
 
-[build-system]
-requires = ["setuptools", "setuptools_scm[toml]>=6.2", "wheel"]
-
-[tool.setuptools.packages.find]
-include = ["greetings*"]
+[tool.hatch.build.targets.sdist]
+include = [
+  "greetings/",
+]
 
-# Add setuptools_scm if you need to generate version numbers from the git hash
-#[tool.setuptools_scm]
 
 # %% [markdown]
 # When installing the package now, pip will also install the dependencies automatically.
 
 # %% language="bash"
 # cd greetings_repo
-# pip install -e .
+# pip install .
```

Comment:
You don't like the `-e` or is not needed with hatch? If we are changing the library, `e` gives us (or used to) an editable library.

Reply:
`-e` is introduced at the end of the lesson, but was somehow used before its introduction 😬 

All the other installations blocks after this block and before the introduction of editable installs don't use `-e`.

Reply:
Oh, good catch! then I'm happy as it is!!! :raised_hands: